staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go

```diff
@@ -52,9 +52,27 @@ const (
 //
 // So that we can recognize master nodes in consequent places though (such as
 // kubectl get nodes), we encourage installations to use the well-known labels.
-// We define NodeLabelRole, which is the preferred form, but we will also recognize
-// other forms that are known to be in widespread use (NodeLabelKubeadmAlphaRole).
+//
+// We define LabelNodeRoleMaster and LabelNodeRoleNode, which are the preferred
+// form, but we will also recognize other forms that are known to be in
+// widespread use (NodeLabelKubeadmAlphaRole, NodeLabelRole, etc) .
+
+const (
+	// LabelNodeRoleMaster is set to the value "true" indicating a master node.
+	// A master node typically runs kubernetes system components and will not typically run user workloads.
+	// When set to any value other than "true" it is ignored.
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
And now we have: `NodeLabelRoleMaster` and `LabelNodeRoleMaster`. At least, I am confused now :-) If one is the successor of the other, maybe let's rename the old one to something with `Legacy`.